.github/copilot-instructions.md

```diff
@@ -38,14 +38,46 @@ This repository contains a Model Context Protocol (MCP) server that drives Commo
 ## Prompts & Personas
 
 - Agent context layers: `data/context/bootstrap.md` → `AGENTS.md` → `.github/prompts/*.prompt.md` → `data/context/chat.md` → RAG fetches. Respect existing tone, persona descriptions, and instructions.
-- MCP tools reside in `src/index.ts`; regenerate the manifest after adding/modifying tool decorators.
+- MCP server wiring lives in `src/mcp-server.ts` (imported by `src/index.ts`). No manifest regeneration step is needed; clients discover tools dynamically via MCP.
 
 ## Review Checklist
 
 1. Tests added/updated and green (`npm test`).
-2. `npm run build` cleanly rebuilds (emits `dist/`, `mcp-manifest.json`).
+2. `npm run build` cleanly rebuilds (emits `dist/`).
 3. Docs amended when user-facing behavior changes.
 4. Commit messages are short, clear, and follow Conventional Commits with clear feat/fix/docs/chore/build/style/refactor/test prefixes.
 5. Code adheres to TDD mindset, KISS, DRY, and maintainability goals.
 
-Thanks for helping keep the Commodore 64 MCP server robust and user-friendly! 
\ No newline at end of file
+Thanks for helping keep the Commodore 64 MCP server robust and user-friendly! 
+
+---
+
+## Using MCP with Copilot Chat (and Eager C64 Activation)
+
+Use these rules to ensure Copilot reliably engages the `c64-mcp` server when the user asks for C64 work.
+
+- Preferred transport is stdio. VS Code should have the experimental MCP setting enabled and a server entry for `c64-mcp` (see `doc/MCP_SETUP.md`).
```

Comment:
@cursoragent Only transport available is stdio. It must always be used. If no c64-mcp server is running it must be started.